analysis/priestdiscipline/src/Guide.tsx

```diff
@@ -0,0 +1,32 @@
+import SPELLS from 'common/SPELLS';
+import { SpellLink } from 'interface';
+import { Section, GuideProps, SubSection } from 'interface/guide';
+
+import type CombatLogParser from './CombatLogParser';
+import { EvangelismApplicators } from './modules/problems/ramps';
+
+export default function Guide({
+  modules,
+  events,
+  info,
+}: GuideProps<typeof CombatLogParser>): JSX.Element {
+  // console.log(modules.ramps.analyzeRamps);
```

Comment:
I like to add a `const debug = false` at the top and then leave select debug logging statements in like `debug && console.log( ...`. This way you can avoid spamming production, but if there's a problem somewhere down the road all you have to do is change the flag on your local.